client/js/lounge.js

```diff
@@ -510,15 +520,18 @@ $(function() {
 				text: "/kick " + itemData,
 			});
 		},
+		download_logs: function(itemData) {
+			socket.emit("logs:download", {
+				name: itemData,
+			});
+		},
 	};
 
-	contextMenuActions.execute = (name, ...args) => contextMenuActions[name] && contextMenuActions[name](...args);
```

Comment:
Any reason you decided to rip this out? You have left it less flexible for any future changes.

Reply:
Also this was made a command pattern specifically because of a review done on a early PR...

Reply:
For the record if someone stops by, we discussed this in the channel. There is nothing definitive here and I'm not planning on breaking anything :) This PR as a whole is WIP.